### DISCUSSION TITLE
Is it possible to run different types of progress bars in parallel?

### DISCUSSION THREAD

**Discussion by ridolfa** [Textualize/rich#1413] [Q&A] [answered]
Hi,
I do my first steps with the excellent rich library .
For a simple application I want to have an inner and outer loop with two
different progress bars. A simple one with only a TextColumn()  and the other with BarColumn()
and more.  The code below works with two identical bars ( progress ) , but so far I couldn't manage to get it working with two different bars ( progress_full , progress_simple ). May be I have overlooked something in the docs and examples. 
Can someone please give me a hint how to get the desired result?

     import time
     from rich.progress import Progress, SpinnerColumn, BarColumn, TextColumn

     filelines = 10
     steplines = 5
     data = "Hello World"
     progress = Progress("{task.description}", TextColumn("[progress.percentage]  task.percentage:>3.0f}%   "+ data),)`

     #progress_simple = Progress("{task.description}", TextColumn("[progress.percentage]{task.percentage:>3.0f}%   "+ data),)
     #progress_full = Progress("{task.description}", SpinnerColumn(), BarColumn(), TextColumn("[progress.percentage]{task.percentage:>3.0f}%   "+ data),)

     total_progress = progress.add_task(description="total", total=filelines, start=True,)
     step_progress = progress.add_task(description="steps", start=False, )

     with progress:

           for i in range(filelines):    #outer loop
             progress.update(total_progress, completed=i)
             progress.reset(step_progress, total=steplines, description=f"steps [bold yellow]#{i}",)
             progress.start_task(step_progress)

           for j in range(steplines):  #inner loop
              progress.update(step_progress, completed=j)
              time.sleep(0.1)
`

**Comment by boegel** [comment, marked as answer]
You need to create your own `Live` instance, and combine the two different `Progress` instances through a `RenderGroup` instance, see the example in #1500

**Comment by ridolfa** [comment]
Thanks. 
I have changed to a GUI (remi) but will try it.
